teamscale-client/src/main/java/com/teamscale/client/PrioritizableTest.java

```diff
@@ -13,6 +15,18 @@
 	/** The reason the test has been selected. */
 	public String selectionReason;
 
+	/** Partition of the test. */
+	public String partition;
+
+	/** Duration in ms. May be null if not set. */
```

Comment:
I was wondering why it would not be set. I think this happens if the duration is not reported during the upload of the test-wise coverage, right? Should we mention this here explicitly so users that get `null` know what they have to do to get a sensible value here?

Reply:
Makes sense.